components/Role.jsx

```diff
@@ -0,0 +1,39 @@
+import FieldsetGroup from './FieldsetGroup';
+import FilterToggle from './FilterToggle';
+
+export default function Availability() {
+  return (
+    <FieldsetGroup legend="role">
+      <FilterToggle
+        type="checkbox"
+        label="Developers"
```

Comment:
I'm guessing these don't have constants already declared? I think that's ok since they only seem to be used once 👍